Sources/CvCityAI.cpp

```diff
@@ -232,6 +232,7 @@ void CvCityAI::AI_doTurn()
 		AI_stealPlots();
 	}
 
+	AI_markBestBuildValuesStale();
```

Comment:
Pretty sure it is not needed to mark it as stale every turn for all cities.
It is marked as stale for each building that is built or removed, for each culture level change, and every time an improvement change inside the city's workable radius, every time a civic is changed, and for each tech invented.
That should be more than often enough.

Reply:
You are very probably right, It was part of an early attempt to see if i could find an improvement, it didnt fly that well, but i havent removed it yet.